content/case_histories/case_histories.json

```diff
@@ -82,5 +82,19 @@
             {"uid":"Mining", "as":"application"},
             {"uid":"Australia", "as":"location"}
         ]
+    },
+    "rockglacier":{
+        "title":"3D Helicopter GPR surveying a rock glacier",
+        "source":"./rockglacier/",
+        "thumbnail":"./rockglacier/images/furggwanghorn_heligpr.png",
+        "citations":[" "],
```

Comment:
If there is no citation, there is no need to include it as an entry. It will be easier to parse later on if it is not there, so could you please remove it?